virttest/qemu_devices/qcontainer.py

```diff
@@ -81,6 +81,7 @@ class DevContainer(object):
     BLOCKDEV_VERSION_SCOPE = '[2.12.0, )'
     SMP_DIES_VERSION_SCOPE = '[4.1.0, )'
     SMP_CLUSTERS_VERSION_SCOPE = '[7.0.0, )'
+    FLOPPY_DEVICE_VERSION_SCOPE = '[6.1.0, )'
```

Comment:
A deeper investigation of the git tree shows that the deprecation commit happened before `v5.1.0-rc0` and in particular:
```
Author: Markus Armbruster <armbru@redhat.com>  2020-06-22 12:42:17
Committer: Markus Armbruster <armbru@redhat.com>  2020-06-23 17:07:07
Parent: fed2c1731c3dfdcf2d69a3ba73da01a80950bcfd (fdc: Open-code fdctrl_init_isa())
Child:  63d5dfbe0de3b021f285a471db632381561e1efb (docs/qdev-device-use.txt: Update section "Default Devices")
Branches: master, remotes/origin/master, remotes/origin/stable-6.0, remotes/origin/stable-6.1, remotes/origin/stable-7.2, remotes/origin/stable-8.0, remotes/origin/staging, remotes/origin/staging-7.2, remotes/origin/staging-8.0
Follows: v5.0.0
Precedes: v5.1.0-rc0

    fdc: Deprecate configuring floppies with -global isa-fdc
    
    Deprecate
    
        -global isa-fdc.driveA=...
        -global isa-fdc.driveB=...
    
    in favour of
    
        -device floppy,unit=0,drive=...
        -device floppy,unit=1,drive=...
    
    Same for the other floppy controller devices.
    
    Signed-off-by: Markus Armbruster <armbru@redhat.com>
    Acked-by: John Snow <jsnow@redhat.com>
    Message-Id: <20200622094227.1271650-7-armbru@redhat.com>
```
So I would say 5.1.0 is the right version to check instead of both 6.1.0 and 6.0.0.